### ISSUE TITLE
Buttons using anchor element have underlined text

### ISSUE THREAD

**Issue by erwstout** [elm-street-technology/elevate-ui#102]
![screen shot 2018-09-27 at 2 49 31 pm](https://user-images.githubusercontent.com/9441913/46176539-a543c400-c264-11e8-80e1-dafd7c3b4ee4.png)

Found this when adding to sites. Can be replicated by setting up the component the following way:

```js
              <Button
                element="a"
                href={`/listings?${savedSearch.searchParameters &&
                  querystring.stringify(savedSearch.searchParameters)}`}
                isOutlined
                color="secondary"
                icon={<Search />}
              >
                View Search
              </Button>
```

We should ensure that the button css is enforcing `text-decoration: none`.

**Comment by erwstout** [comment]
PR submitted: https://github.com/elm-street-technology/elevate-ui/pull/103

**Comment by chrisheninger** [comment]
👏  Thank you!

Released in v0.14.5
